src/scores/categorical/contingency_impl.py

```diff
@@ -136,6 +136,17 @@ def probability_of_detection(self):
 
         return pod
 
+    def false_alarm_ratio(self):
+        """
```

Comment:
I don't see a reference (or have I overlooked it?)

Reply:
Added